README.md

```diff
@@ -15,11 +15,20 @@ You can download it here:
 
 Currently it's necessary to unsecure the management http-interface to be able to run testsuite.
 E.g. for standalone Wildfly 10 using CLI:
+
+### Standalone
+
```

Comment:
Above this title it stands `E.g. for standalone Wildfly 10 using CLI:`. The title should be more properly incorporated with the previous text :-)